.vscode/launch.json

```diff
@@ -58,7 +58,7 @@
                 "NBXPLORER_URI": "http://127.0.0.1:32838",
                 "NBXPLORER_BTCRPCUSER": "polaruser",
                 "NBXPLORER_BTCRPCPASSWORD": "polarpass",
-                "NBXPLORER_BTCRPCURL": "http://127.0.0.1:18443/",
```

Comment:
I think we are having a clash here @RodriFS, I have to change this all the time

Reply:
This change was done because polar's bitcoind defaults to 18445, and we can use the polar UI to fund and create channels with the `docker-compose.dev-novs.yml` file. What docker compose are you using? Maybe we can also change that compose file to use 18445

Reply:
Changed back